### DISCUSSION TITLE
HTML Report renders fine in some WB and not others

### DISCUSSION THREAD

**Discussion by SwearyStats** [datapane/datapane#129] [Help]
I've been encountering a strange behaviour with the rendering of the report in certain circumstances. I use report.save() and most web browsers such as Chrome/Firefox etc. work perfectly fine, including on Android, Linux etc. platforms. 

The issue is that when I load this on my work computer the report only has its html components loaded, none of the graphs or interactive parts load. I thought this was due to a VPN issue and JS resource callouts so I disconnected the internet, wiped the browser and tried again but the same result occurred. I tried multiple browsers, all with the same result.

I was wondering if anyone could advise on this?

p.s. This is an amazing package btw!

**Comment by mands** [comment]
Hi @SwearyStats ,

That's interesting - could you provide some further details on your work computer - e.g. the OS and Web Browsers (and the versions) you tried? I don't think it would be a network issue - locally saved reports include everything in the bundle and don't require network access.

Generally we test on Chrome, Firefox and Safari, mainly on Linux and macOS. It could perhaps be an unsupported browser, such as IE, or maybe an older version of Chrome that is upgraded via an admin policy?
